__tests__/app.test.js

```diff
@@ -287,3 +287,72 @@ describe('api/articles/:article_id/comments', () => {
     });
   });
 });
+
+describe('api/articles/:article_id/comments', () => {
+  describe('POST- Successful Responses', () => {
+    test('status:201 and endpoint responds with the added restaurant object', () => {
```

Comment:
Please don't copy and paste test blocks without proof reading...there should be no restaurants in this repo